README.md

```diff
@@ -0,0 +1,214 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The following technologies have been used for the project development.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Rails.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The key features of the application are:
+
+- **[Create and delete books]()**
+- **[List books]()**
+- **[Select book]()**
+- **[Filter books]()**
+- **[Create and delete users]()**
+- **[List users]()**
+- **[Select user]()**
+- **[Filter users]()**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](#) (Coming soomn)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:diarisdiakite/School-library
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd School-library
+  gem install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rails test test/models/School-library_test.rb
+```
+
+### Deployment
+
+You can deploy the School-library application using: google cloud, Microsoft Azure, Netlify, github, render and other platform by giving credit on using our template and/or contribute via pull requests (read more in the contributing section).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Didy
+
+GitHub: [@Didy](github.com/diarisdiakite)
+<br> Twitter: [@diarisdiakite](https://www.twitter.com/diarisd)
+<br>LinkedIn: [@diarisdiakite](https://www.linkedin.com/in/diariatou-diakite-67ab80165/)
+<br><br>
+
+[`Back to top`](#readme-top)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Dynamic list creating by the users using the interface]()**
```

Comment:
[OPTIONAL]
- [ ] I kindly advice that you do not make these future features as links and you can do this by removing the parentheses`()` in front of the texts :+1: :sparkles: